hls4ml/writer/quartus_writer.py

```diff
@@ -509,18 +509,21 @@ def write_activation_tables(self, model):
         h_file.close()
 
         ###################
-        ## sigmoid_table
+        # sigmoid_table
         ###################
+        CENTERED = True
```

Comment:
Can you remind me if there is any reason to keep the non-centered versions? There seems to be no advantage to that. If so, the non-centered code path should be removed.

Reply:
I had left it in case someone would like to use or test the non-centralized functionality, but particularly I don't see any advantages in using it. I will remove it so.